extensions/amp-story-page-attachment/0.1/amp-story-open-page-attachment.js

```diff
@@ -128,13 +132,21 @@ const renderOutlinkUI = (pageEl, attachmentEl) => {
   // Set image.
   const openImgAttr = attachmentEl.getAttribute('cta-image');
 
+  const noAnimation =
+    getExperimentBranch(getWin(pageEl), 'story-ad-auto-advance') == 'c';
```

Comment:
Where is this constant `c` from? Do you need to replace it with the correct value from `StoryAdSegmentExp.AUTO_ADVANCE_NEW_CTA_NOT_ANIMATED`?